drag_and_drop_v2/utils.py

```diff
@@ -78,3 +78,5 @@ def not_placed(number, ngettext=ngettext_fallback):
 
 
 FeedbackMessage = namedtuple("FeedbackMessage", ["message", "message_class"])  # pylint: disable=invalid-name
+ItemStats = namedtuple('ItemStats', ["required", "placed", "correctly_placed",  # pylint: disable=invalid-name
+                                     "decoy", "decoy_in_bank"])
```

Comment:
Nit: The edX Python Style Guide does not explicitly mention lists, but I believe we try to use the same line-breaking rules as for function calls, so this should be formatted as:

```
ItemStats = namedtuple(
    'ItemStats',
    ["required", "placed", "correctly_placed", "decoy", "decoy_in_bank"]
)
```